tests/test_regression.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+import requests
+import tarfile
+import json
+import os
+import pyhf
+import numpy as np
+
+
+@pytest.fixture(scope='module')
+def sbottom_likelihoods_download():
+    """Download the sbottom likelihoods tarball from HEPData"""
+    sbottom_HEPData_URL = "https://www.hepdata.net/record/resource/997020?view=true"
+    targz_filename = "sbottom_workspaces.tar.gz"
```

Comment:
should we compare some download checksums here?

Reply:
This is a good idea.

Reply:
@lukasheinrich Done.